(GH-2747) Default to showing all targets with 'inventory show' (#2775)
puppetlabs/bolt: guides/targets.txt

TOPIC
    targets

DESCRIPTION
    A target is a device that Bolt connects to and runs actions on. Targets can
    be physical, such as servers, or virtual, such as containers or virtual
    machines.

>     Several of Bolt's commands connect to targets and run actions on them. For
>     example, the `bolt script run` command and `Invoke-BoltScript` PowerShell
>     cmdlet run a script on a target. When you run these commands, Bolt requires
>     a list of targets to run the action on. You can provide this list of targets
>     using one of the following command-line options:

Reviewer: ```suggestion
    Several of Bolt's commands connect to targets and run actions on them. When
    you run these commands, Bolt requires a list of targets to run the action on. For
    example, the `bolt script run` command and `Invoke-BoltScript` PowerShell
    cmdlet require you to provide a list of targets to run your script on. You can 
    provide a list of targets to a command using one of the following 
    command-line options:
```

TOPIC
    targets

DESCRIPTION
    A target is a device that Bolt connects to and runs actions on. Targets can
    be physical, such as servers, or virtual, such as containers or virtual
    machines.

    Several of Bolt's commands connect to targets and run actions on them. For

>     example, the `bolt script run` command runs a script on a target. When you

Reviewer 2: ```suggestion
    example, the `bolt script run` command and `Invoke-BoltScript` PowerShell cmdlet run a script on a target. When you
```

    run these commands, Bolt requires a list of targets to run the action on.
    You can provide this list of targets using one of the following command-line
    options:

>         -t, --targets TARGETS

Reviewer 2: This feels more like reference material than a guide. I think it'd be tidier to list the options (possibly even generate them from `lib/bolt/cli.rb#L266`) and then either link to reference docs or instruct to run `bolt <subcommand> --help` to see details. Defer to @hestonhoffman if he thinks the expanded version will be more helpful.

Author: Removed the description of each and just left the options themselves in.
